CHANGELOG.md

```diff
@@ -120,6 +120,7 @@
 * [ENHANCEMENT] Rollout-operator: upgrade to v0.14.0.
 * [ENHANCEMENT] Add `_config.autoscaling_querier_predictive_scaling_enabled` to scale querier based on inflight queries 7 days ago. #7775
 * [ENHANCEMENT] Add support to autoscale ruler-querier replicas based on in-flight queries too (in addition to CPU and memory based scaling). #8060 #8188
+* [ENHANCEMENT] Distributor: improved distributor HPA scaling metric to only take in account ready pods. #8250
```

Comment:
Is it worth noting that users must be scraping `kube_pod_status_ready` in their keda prometheus?

Reply:
Yes, it a good feedback. Adding it.

Reply:
Done in https://github.com/grafana/mimir/pull/8251/commits/329aef900c6729877a96685f7d02e8c0bbc9754e